app/src/main/res/values-nl/strings-nl.xml

```diff
@@ -7,21 +7,21 @@
 	<string name="menu_stop">Stop</string>
 	<string name="menu_refresh">Verversen</string>
 	<string name="menu_export_database">Export databank</string>
```

Comment:
To stay consistent, I would change this to "Exporteer database" (I guess databank is valid Dutch, but I have never heard anybody use it)